src/main/utils/validation.ts

```diff
@@ -0,0 +1,4 @@
+
+export const isFieldFilledIn = (value: string): (boolean | string)[] => {
+  return [!!value, 'required'];
```

Comment:
I don't think this will work the way I imagined.

I think it should return false if there are no errors or the error type if there is an error. That allows you to chain together multiple checks:

```
validation: field => isFieldFilledIn(field) || isFieldLongerThan(field, 3) || isFieldShorterThan(field, 255)
```

The first one that returns something truthy (e.g. 'required') would be the return value and be the error.